### PR TITLE
More consistent README.md

### PR DESCRIPTION
**Pull request by L11R**
Sorry, I am perfectionist...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
+On Debian/Ubuntu:
 ```bash
-apt install build-essential libssl-dev
+apt install build-essential libssl-dev zlib1g-dev
 ```
-
-On CentOS:
+On CentOS/RHEL:
 ```bash
 yum install openssl-devel
```
Comment (stek29) [inline README.md]: mind adding zlib-devel?

Comment (L11R) [reply README.md]: @stek29 done.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,20 @@
 # MTProxy
 
-# Building
+## Building
+Install dependencies, you would need common set of tools for building from source, and source packages for `openssl` and `zlib`.
```
Comment (stek29) [inline README.md]: Why "source packages" and not "development packages"? I think source packages stands for getting actual source from which the package was built from.

Comment (L11R) [reply README.md]: @stek29 Debian site call them so: 
image
I can rename it...

Comment (stek29) [reply README.md]: openssl is the source package, not the libssl-dev, no?
and libssl-dev seems to be called "development files" too...

Comment (L11R) [reply README.md]: @stek29 okay, I fixed it.

**Pull request merged**